packages/react-composites/src/composites/MeetingComposite/NotificationIcon.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { Icon, Stack } from '@fluentui/react';
+import { NotificationIconStyles, NotificationTextStyles } from './styles/MeetingCompositeStyles';
+
+/**
+ * @private
+ */
+export type NotificationIconProps = {
+  numberOfMessages: number;
+};
+/**
+ * @private
+ */
+export const NotificationIcon = (props: NotificationIconProps): JSX.Element => {
+  const { numberOfMessages } = props;
+  return (
+    <Stack horizontalAlign="center" verticalAlign="center" style={{ position: 'absolute', top: 0, right: '0.78rem' }}>
```

Comment:
can we avoid using inline styles unless absolutely necessary?

Moving to another file would be my suggestion.

Reply:
I think it's important to understand the reasoning behind why we do somethings the way we do them.
For example, a common PR comment I see from many team members is the suggestion of moving an inline style to another a constant in another file.
Which to me personally has no advantage (apart from peoples personal preference around readability).

If we were to not use `style` prop and use `className` instead with `mergeStyles`, that presents an advantage I can clearly see. For example, re-use of some class names, ability to compress css files etc.

Reply:
Please do extract this particular one into a constant. Because that positioning is quite tricky -- it's trying to be just above the icon used by the button this overlays. This context is not at all obvious from looking at the `style` and deserves a detailed comment in the code.

Reply:
this is resolved now in this PR: https://github.com/Azure/communication-ui-library/pull/1378